hscontrol/app.go

```diff
@@ -245,11 +245,11 @@ func (h *Headscale) scheduledTasks(ctx context.Context) {
 
 	lastExpiryCheck := time.Unix(0, 0)
 
-	derpTicker := time.NewTicker(h.cfg.DERP.UpdateFrequency)
-	defer derpTicker.Stop()
-	// If we dont want auto update, just stop the ticker
-	if !h.cfg.DERP.AutoUpdate {
-		derpTicker.Stop()
+	// Only start the ticker if auto update is enabled and has nonzero duration
+	var derpTicker *time.Ticker
```

Comment:
This will panic later when `derpTicker.C` is called in the `select`, I've fixed this in #2368